packages/web-components/src/components/va-radio-option/va-radio-option.css

```diff
@@ -1,34 +1,39 @@
-@import '../../mixins/inputs.css';
-
 :host {
+  --visible-radio-button-size: 1.4rem;
   display: block;
   margin-top: 1rem;
 }
 
+input {
+  height: var(--visible-radio-button-size);
+  margin: 0.5rem 0 0 2.4rem;
+  opacity: 0;
+  position: absolute;
+  width: var(--visible-radio-button-size);
+}
+
 label {
-  box-sizing: border-box;
-  outline: none;
-  display: inline-block;
-  margin: 0;
-  width: 100%;
-  padding-left: 2.4rem;
+  margin-left: 2.4rem;
 }
 
 label::before {
-  box-sizing: border-box;
+  background-color: var(--color-white);
   border-radius: 100%;
   box-shadow: 0 0 0 2px var(--color-white), 0 0 0 3px var(--color-gray-medium);
-  height: 1.4rem;
-  line-height: 1.4rem;
-  margin-right: 1rem;
-  width: 1.4rem;
-  background: var(--color-white);
-  background-color: var(--color-white);
   content: '\A0';
   display: inline-block;
+  height: var(--visible-radio-button-size);
+  line-height: var(--visible-radio-button-size);
+  margin-right: 1rem;
+  width: var(--visible-radio-button-size);
+}
+
+input:not([disabled]):focus + label::before {
+  outline: 2px solid var(--color-gold-light);
+  outline-offset: 6px;
```

Comment:
This looks like the "main" change, is that right? I was able to get what looks like the same styling without changing all of the other styling & markup, so do we need the rest of the changes?

![focus styling only applied to button](https://user-images.githubusercontent.com/2008881/168392594-8e573f74-5ec7-4d84-a594-02ccd413b42c.png)

Reply:
You're correct. We don't need any of those other changes. Thanks! I'm closing this PR for a new one since I had issues reverting.